src/config/cluster.rst

```diff
@@ -85,3 +85,39 @@ Cluster Options
 
         [cluster]
         seedlist = couchdb@node1.example.com,couchdb@node2.example.com
+
+RPC Performance Tuning
+======================
+
+.. config:section:: rexi :: Internal RPC Tuning
+
+    CouchDB uses distributed Erlang to communicate between nodes in a cluster.
+    The ``rexi`` library provides an optimized RPC mechanism over this
+    communication channel. There are a few configuration knobs for this system,
+    although in general the defaults work well.
+
+    .. config:option:: buffer_count
+
+    The local RPC server will buffer messages if a remote node goes unavailable.
+    This flag determines how many messages will be buffered before the local
+    server starts dropping messages. Default value is ``2000``.
+
+    .. config:option:: server_per_node
+
+    Early versions of this RPC library used a single local Erlang process for
+    sending messages to all other nodes in a cluster. The current default is to
+    spawn a local gen_server process for each node in the cluster. It is not
+    recommended to change this setting from the default.
```

Comment:
Is this history lesson helpful to CouchDB users? Could we simplify this paragraph with something like:

> By default, ``rexi`` will spawn one local ``gen_server`` process for each node in the cluster. This setting changes the number of ``gen_server`` processes per node. It is not recommended to change this setting from the default.

Reply:
Good call. I'm sitting here telling the user why she should ignore this option, and wasting even more of her time in doing so! I took another pass with essentially your recommendation.